### PR TITLE
add #include <memory> to ibtk solver_utilities

### LINKED ISSUE #1359: Missing #include <memory> in ibtk solver_utilities
Master fails to compile on Linux systems with `error: 'unique_ptr' is not a member of 'std'`

### PR DESCRIPTION
**Pull request by jabrown893**
Fixes #1359 

### IBAMR Pull Request Checklist
- [x] Does the test suite pass?
- [x] Was clang-format run?
- [x] Were relevant issues cited? Please remember to add `Fixes #12345` to close
      the issue automatically if we are fixing the problem.
- [x] Is this a change others will want to know about? If so, then has a
      changelog entry been added?
- [x] If new data structures have been added to a class then have they been set
      up appropriately for restarts? If so, ensure that the restart version
      number is incremented.
- [x] Does this change include a bug fix or new functionality? If so, a new test
      or tests should be added. New tests should run quickly (less than a minute
      in release mode). If possible, an older test should gain a new option so
      that we do not need to compile more test executables.
- [x] Did you (if your account has permission to do so) set relevant labels on
      GitHub for the pull request?

### CODE REVIEW

File: ibtk/src/solvers/solver_utilities.cpp [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 /////////////////////////////// INCLUDES /////////////////////////////////////
 
+#include <memory>
 #include "ibtk/solver_utilities.h"
```
Comment (drwells) [inline ibtk/src/solvers/solver_utilities.cpp:18]:
```suggestion
#include "ibtk/solver_utilities.h"

#include <memory>

```

system headers need to come after IBTK (or IBAMR) headers. clang-format will fix this automatically for you.

**Comment by jabrown893** [comment]
@drwells Do you want me to cleanup history here? If so, can you remind me how to do that?

**Comment by drwells** [comment]
Its not critical but if you have time it would be nice. One way to clean things up would be to undo the commits (but not the changes) with something like
```
git checkout unique_ptr
git reset --soft master
git commit -m "add a missing header."
```
and then do a force push.

**Comment by jabrown893** [comment]
Ah, yes. Force push - that's what I couldn't remember.

**Review by drwells — APPROVED** [review]

**Pull request merged**